smsjwplatform/jwplatform.py

```diff
@@ -98,6 +98,28 @@ def key_for_media_id(media_id, preferred_media_type='video', client=None):
     return video_resource['key']
 
 
+def get_acl(key, client=None):
+    """
+
+    :param key: JWPlatform key for the media.
+    :param client: (options) an authenticated JWPlatform client as returned by
+        :py:func:`.get_jwplatform_client`. If ``None``, call :py:func:`.get_jwplatform_client`.
+    """
+    client = client if client is not None else get_jwplatform_client()
+    try:
+        video = client.videos.show(video_key=key)
+        field = video.get('video', {}).get('custom', {}).get('sms_acl', None)
+        field_parts = field.split(":")
+        assert (
+            len(field_parts) == 3 and
+            field_parts[0] == 'acl' and
+            field_parts[2] == ''
+        ), "sms_acl should be of the format 'acl:{ACL}:"
```

Comment:
Firstly, I think this should be ``f"sms_acl...``, not ``"sms_acl...`` but also I feel this is a prime use case for the re module. Something like:
```python
# ... at top of module ...

#: regex for parsing video metadata
METADATA_RE = re.compile(r'^([a-z][a-z0-9_]):(.*):$')

def parse_custom_field(expected_type, field):
    """
    Parses a custom field content of the form "<type>:<value>:". Returns the value tuple.
    Raises ValueError if the field is of the wrong form or has wrong type.

    """
    match = METADATA_RE.match(field)
    if not match:
        raise ValueError(f'Field has invalid format: {field}')
    field_type, value = match.groups()
    if fied_type != expected_type:
        raise ValueError(f'Field has unexpected type "{field_type}". Expected "{expected_type}".')
    return value

# ... in get_acl() ...
return parse_custom_field('acl', field)
```
The parse_custom_field() function is then re-usable since we're definitely going to want to do this going forward.